### ISSUE TITLE
more versions

### ISSUE THREAD

**Issue by DARKWORR1OR** [bigreefer/TheMissingVillages#1]
I would like you devs to make more versions for this amazing mod..
As we would all know most of the Minecraft community player have potato pc's , therefore I would suggest you guys to make a version on 1.18

**Comment by xiaoyin2011** [comment]
Yes, I hope that the mod that attracts me could be updated to version 1.18.2
